webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ParametersApi.scala

```diff
@@ -481,7 +481,9 @@ class ParameterApiService14(
   }
 
   def listParameters(): IOResult[Seq[JRGlobalParameter]] = {
-    readParameter.getAllGlobalParameters().map(_.sortBy(_.name).map(JRGlobalParameter.fromGlobalParameter(_, None)))
+    readParameter
+      .getAllGlobalParameters()
+      .map(_.filter(_.visibility == Visibility.Displayed).sortBy(_.name).map(JRGlobalParameter.fromGlobalParameter(_, None)))
```

Comment:
Same question about filtering inside `JRGlobalParameter.fromGlobalParameter(...)` instead

Reply:
same answer, it's sooner here.